cosmwasm/main.go

```diff
@@ -0,0 +1,280 @@
+package main
+
+import (
+	"embed"
+	"encoding/gob"
+	"fmt"
+	"io"
+	"io/fs"
+	"io/ioutil"
```

Comment:
The `ioutil` package is deprecated so it should be removed:

```suggestion
```

There are other suggestions to replace `ioutil` calls by `os` ones.

Reply:
resolved https://github.com/yerasyla/plugins/commit/548b8fc1967aebdc6bd0eb85d334b2cb6f599992